commons-packet/commons-packet-manager/src/main/java/io/mosip/commons/packet/util/PacketValidator.java

```diff
@@ -308,16 +308,22 @@ private LinkedHashMap loadDemographicIdentity(Map<String, Object> fieldMap) thro
                 if (value != null) {
                     Object json = new JSONTokener(value).nextValue();
                     if (json instanceof org.json.JSONObject) {
-                        HashMap<String, Object> hashMap = new ObjectMapper().readValue(value, HashMap.class);
+                        HashMap<String, Object> hashMap = mapper.readValue(value, HashMap.class);
                         demographicIdentity.putIfAbsent(e.getKey(), hashMap);
                     }
+
                     else if (json instanceof JSONArray) {
                         List jsonList = new ArrayList<>();
+
                         JSONArray jsonArray = new JSONArray(value);
-                        for (int i = 0; i < jsonArray.length(); i++) {
-                            Object obj = jsonArray.get(i);
-                            HashMap<String, Object> hashMap = new ObjectMapper().readValue(obj.toString(), HashMap.class);
-                            jsonList.add(hashMap);
+                        if (jsonArray.length() > 0 && jsonArray.get(0) instanceof String) {
+                            demographicIdentity.putIfAbsent(e.getKey(), jsonArray.toString());
```

Comment:
why should it be jsonArray.toString()? can it just be jsonArray?

Reply:
As per discussion.  jsonArray.toString() is required. if we use jsonArray then ends up with error "failed to parce to json object".